### PR TITLE
Fix chmod 000

### PR DESCRIPTION
**Pull request by TemKaSD**
Fix chmod 000 in files appParameters.php and class_index.php.
https://github.com/PrestaShop/docs/issues/308

### CODE REVIEW

File: src/content/1.7/scale/optimizations.md [hunk 1]
```diff
@@ -101,6 +101,8 @@ opcache.enable_file_override = On
 opcache.interned_strings_buffer=64
 opcache.memory_consumption=256
 opcache.max_accelerated_files=20000
+opcache.validate_timestamps=1
```
Comment (PierreRambaud) [inline src/content/1.7/scale/optimizations.md]: This is the default value so not needed.

File: src/content/1.7/scale/optimizations.md [hunk 2]
```diff
@@ -101,6 +101,8 @@ opcache.enable_file_override = On
...
+opcache.revalidate_freq=0
```
Comment (PierreRambaud) [inline src/content/1.7/scale/optimizations.md]: Not sure it's a good idea to revalidate cache on each request :thinking:

Comment (TemKaSD) [reply src/content/1.7/scale/optimizations.md]: > Not sure it's a good idea to revalidate cache on each request 🤔

Without these changes error 500, check.

Comment (TemKaSD) [reply src/content/1.7/scale/optimizations.md]: @PierreRambaud https://youtu.be/5rdthNnruFE

Comment (PierreRambaud) [reply src/content/1.7/scale/optimizations.md]: I'm not able to reproduce with nginx / apache2.
Are you sure your php have the same user as your web server? Which user / group have your ftp account?

Comment (PierreRambaud) [reply src/content/1.7/scale/optimizations.md]: If these configuration lines work for you, you can keep it, and if you have a problem we can't reproduce, we're not able to approve changes. 
But, if you can describe a little more in a troubleshooting section what you had, and how you fixed it (by adding these two lines), we'll be glad to approve it :)

Comment (TemKaSD) [reply src/content/1.7/scale/optimizations.md]: > I'm not able to reproduce with nginx / apache2.
> Are you sure your php have the same user as your web server? Which user / group have your ftp account?

Yes.
phpfpm:

user = mbm
group = mbm
listen.owner = mbm
listen.group = www-data

files:

user = mbm
group = mbm

Comment (TemKaSD) [reply src/content/1.7/scale/optimizations.md]: > If these configuration lines work for you, you can keep it, and if you have a problem we can't reproduce, we're not able to approve changes.
> But, if you can describe a little more in a troubleshooting section what you had, and how you fixed it (by adding these two lines), we'll be glad to approve it :)

Link pls troubleshooting section.

Comment (PierreRambaud) [reply src/content/1.7/scale/optimizations.md]: You need to create one :)

Comment (TemKaSD) [reply src/content/1.7/scale/optimizations.md]: > You need to create one :)

?

Comment (PierreRambaud) [reply src/content/1.7/scale/optimizations.md]: There is no Troubleshooting section here, you need to create a new one. Can be at the bottom of this file.

**Pull request closed without merging**